### DISCUSSION TITLE
why a prediction with small iou to ground truth has higher confidence

### DISCUSSION THREAD

**Discussion by xoofee** [ultralytics/yolov5#10076] [General]
000000006771.jpg from coco 2017

python detect.py --weight yolov5x.pt --source coco-2017/data/000000006771.jpg

see the person first on the left, a small prediction box has confidence 0.67
![image](https://user-images.githubusercontent.com/7467604/200547522-b047c75e-2aba-45f1-a3c6-134115c4d155.png)

while a larger prediction box has confidence 0.27
![image](https://user-images.githubusercontent.com/7467604/200547565-510e50ed-28cb-437d-a4dd-07540ce755b3.png)

the ground truth is
![image](https://user-images.githubusercontent.com/7467604/200548344-2684f64c-fd12-4cb1-8cb9-0aefd4b487d0.png)

**Comment by glenn-jocher** [comment]
@xoofee anecdotal results aren't very useful for trying to establish patterns. I would maybe plot areas or sizes of a population of detections vs ground truths if you are seeking patterns or trends.

**Reply by xoofee** [reply]
I'll try it, thanks!
